recipes/libcds/all/patches/fix-cmake.patch

```diff
@@ -0,0 +1,57 @@
+--- a/CMakeLists.txt
```

Comment:
Has this patch been sent to upstream?

Reply:
No, modifications related to `if(WIN32)`, `PUBLIC CDS_BUILD_STATIC_LIB` and maybe `RPATH` could be submitted to upstream.